pom.xml

```diff
@@ -292,10 +293,10 @@
 
         <!-- Asciidoctor -->
         <hibernate-asciidoctor-theme.version>1.0.1.Final</hibernate-asciidoctor-theme.version>
```

Comment:
While we're at it, let's update the theme to 1.0.2.Final. It should improve the code formatting in the PDF doc.

Reply:
ok will do, but that's a separate issue.

Reply:
Added a commit for that ;)